app/scripts/main.js

```diff
@@ -348,6 +351,14 @@ document.addEventListener("DOMContentLoaded", function () {
             return call(host, "CreateDatabase2", params).then(processResult);
         },
 
+        /**
+         * Get session credentials
+         * @param params {object} - the create database parameters
+         * @returns {object} - the database, user and password
+         */
+        getSessionCredentials = function(){
```

Comment:
This is unused

Reply:
yes true. Did I create it. Ok I will remove it.